plugins/modules/organization.py

```diff
@@ -87,11 +171,50 @@ def main():
             name=dict(required=True),
             description=dict(),
             label=dict(),
+            upstream_type=dict(required=False, choices=['redhat_cdn', 'export_sync', 'network_sync']),
+            upstream_url=dict(required=False),
+            upstream_username=dict(required=False),
+            upstream_password=dict(required=False, no_log=True),
+            # upstream_ca_cert=dict(required=False, type='entity', resource_type='content_credentials', scope=['']),
+            upstream_ca_cert_id=dict(required=False),
```

Comment:
we really shouldn't require the user to provide an `id`, why did the commented code not work? because of the scope?

in other places, we use it like this:
```python
ssl_ca_cert=dict(type='entity', resource_type='content_credentials', scope=['organization']),
```

Reply:
Yes, I fully agree, this is just my current 'worked-around' version, and indeed, because of the scope.

Reply:
@evgeni I think at first I copied that line of code from ```repository.py``` or something like it, but it kept complaining about it :/